packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -0,0 +1,81 @@
+import {
+  Component,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  State,
+  h,
+} from '@stencil/core';
+
+@Component({
+  tag: 'va-additional-info',
+  styleUrl: 'va-additional-info.css',
+  shadow: true,
+})
+export class VaAdditionalInfo {
+  @State() open: boolean;
+
+  /**
+   * The text to trigger the expansion
+   */
+  @Prop() trigger: string;
+
+  /**
+   * If true, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics: boolean;
+
+  @Event({
+    eventName: 'component-library-analytics',
+    composed: true,
+    bubbles: true,
+  })
+  componentLibraryAnalytics: EventEmitter;
+
+  toggleOpen(): void {
+    // Conditionally track the event.
+    if (!this.disableAnalytics) {
+      this.componentLibraryAnalytics.emit({
+        componentName: 'va-additional-info',
+        action: !this.open ? 'expand' : 'collapse',
+        details: {
+          triggerText: this.trigger,
+        },
+      });
+    }
+
+    this.open = !this.open;
+  }
+
+  handleKeydown(event): void {
+    if (event.key === ' ' || event.key === 'Enter') {
+      event.preventDefault();
+      this.toggleOpen();
+    }
+  }
```

Comment:
Expanding on `Enter` is an enhancement over current behavior:

https://github.com/department-of-veterans-affairs/component-library/blob/4a7c522fe2f21d8ba0e7b0bea795fe0d959a2a0c/packages/react-components/src/components/AdditionalInfo/AdditionalInfo.jsx#L15-L20